launcher/src/main.rs

```diff
@@ -65,6 +66,7 @@ fn get_config(
 }
 
 fn resolve_attention(config: &Option<Config>, lora_adapters: &Option<String>) -> (String, String) {
+    let compute_capability = *gpu::COMPUTE_CAPABILITY;
```

Comment:
Why not just `get_cuda_capability()` ?

Reply:
No need to start a slow GIL if it's not needed.

Reply:
I am not sure I understand. This will only evaluate the compute capability once, it's just a cheap Rust lock. If we'd call `get_cuda_capability`, we'd call into Python land every time.

Reply:
Yes yes I understand, but since we're only calling once there's no need, but it's really totally not important.